tools/devops/automation/templates/sign-and-notarized/dotnet-signing.yml

```diff
@@ -48,61 +106,22 @@ steps:
       }
 
       # Write the json to a file
-      ConvertTo-Json -InputObject $SignFileList -Depth 5 | Out-File -FilePath $(Build.ArtifactStagingDirectory)/MsiFiles2Notarize.json -Force
-      dotnet $Env:MBSIGN_APPFOLDER/ddsignfiles.dll /filelist:$(Build.ArtifactStagingDirectory)/MsiFiles2Notarize.json
-    displayName: 'Sign .msi'
-    condition: ${{ parameters.condition }}
-
-- pwsh: |
-    mv $(Build.SourcesDirectory)/package/bundle.zip $(Build.ArtifactStagingDirectory)/not-signed-bundle.zip
-    $bundlePath = "$(Build.ArtifactStagingDirectory)/bundle"
-    unzip $(Build.ArtifactStagingDirectory)/not-signed-bundle.zip -d $bundlePath
-    $patterns = @(
-      "*.iOS.dll",
-      "*.tvOS.dll",
-      "*.Mac.dll", "*.macOS.dll", "*XamMac.dll",
-      "*.MacCatalyst.dll",
-      "*.WatchOS.dll"
-    )
-    $files = @()
-    foreach ($p in $patterns) {
-      $files += Get-ChildItem -Path $bundlePath -Recurse  -Filter $p
-    }
-
-    $SignFiles = @()
-    foreach($f in $files) {
-        Write-Host "$($f.FullName)"
-        $SignFiles += @{ "SrcPath"="$($f.FullName)"}
-    }
-
-    $SignFileRecord = @(
-      @{
-        "Certs" = "400";
-        "SignFileList" = $SignFiles;
-      }
-    )
-
-    $SignFileList = @{
-        "SignFileRecordList" = $SignFileRecord
-    }
-
-    # Write the json to a file
-    ConvertTo-Json -InputObject $SignFileList -Depth 100 | Out-File -FilePath $(Build.ArtifactStagingDirectory)/bundle.json -Force
-    dotnet $Env:MBSIGN_APPFOLDER/ddsignfiles.dll /filelist:$(Build.ArtifactStagingDirectory)/bundle.json
-    # rezip and move back
-    ditto -c -k --sequesterRsrc $bundlePath bundle.zip
-    mv bundle.zip $(Build.SourcesDirectory)/package/bundle.zip
-  displayName: 'Sign bundle.zip'
-  workingDirectory: $(Build.ArtifactStagingDirectory)
-
-- template: publish-nugets.yml
-  parameters:
-    isPR: ${{ parameters.isPR }}
-
-# always upload no matter what, since if we are not signing we need the artifact in the pipeline
-- task: PublishPipelineArtifact@1
-  displayName: 'Publish Notarized Dotnet Artifacts'
-  inputs:
-    targetPath: $(Build.SourcesDirectory)/package
-    artifactName: dotnet-signed
-  continueOnError: true
+      ConvertTo-Json -InputObject $SignFileList -Depth 100 | Out-File -FilePath $(Build.ArtifactStagingDirectory)/bundle.json -Force
+      dotnet $Env:MBSIGN_APPFOLDER/ddsignfiles.dll /filelist:$(Build.ArtifactStagingDirectory)/bundle.json
+      # rezip and move back
+      ditto -c -k --sequesterRsrc $bundlePath bundle.zip
+      mv bundle.zip $(Build.SourcesDirectory)/package/bundle.zip
+    displayName: 'Sign bundle.zip'
+    workingDirectory: $(Build.ArtifactStagingDirectory)
```

Comment:
Is this bundle.zip file included in any of the shipping dotnet nupkgs? If not, can this be signed in a different job? Assuming this and the MSI aren't needed by dotnet nupkgs you may be able to remove this `Sign & Notarize Dotnet` job entirely.

Reply:
These are still needed and used by XMA IIRC

Reply:
Yeah, we need it from XamarinVS to build the Xamarin.iOS vsix that ships the traditional Sdk

Reply:
If `bundle.zip` is only used by the classic SDK/VSIX then presumably the signing of it could be moved into a different job in the future, allowing us to eventually remove this `Sign & Notarize Dotnet` stage entirely to save some more time.